guides/satellite-sync.sh

```diff
@@ -104,6 +104,7 @@ sed 's/:ProductVersionPrevious: 6.6/:ProductVersionPrevious: 6.6/g' common/attri
 mv common/attributes.tmp common/attributes.adoc
 find common -name '*.adoc' -type f -exec sed -i -e 's/{project-context}/satellite/g' -- {} +
 find common -name '*.adoc' -type f -exec sed -i -e 's/{smart-proxy-context}/capsule/g' -- {} +
+find common -name '*.adoc' -type f -exec sed -i -e 's/{OpenStack}/Red{nbsp}Hat OpenStack Platform/g' -- {} +
```

Comment:
Sure: Much like those other replacements in the two lines above, this is here because when setting one attribute (in this case {CRname}) to resolve to another attribute (in this case {OpenStack}), it works fine in the upstream build but fails in the downstream build for some reason. So, like Melanie did with the other two items, I added this line to have the script change the second attribute (the upstream-downstream one) to just its downstream value.

Reply:
Actually, we also have :ProductName: {ProjectServer} which seems to work fine. I might have over-thought this. 

@melcorr do you remember if the double attribute was a problem for downstream builds generally, or only in IDs?

Reply:
Yeah false alarm, I just temporarily copied docinfo.xml over to the upstream guide to test a build with bccutil and it seems fine.

Reply:
Just got the notification now. Inefficient!